assets/js/view/company.rb

```diff
@@ -13,30 +13,71 @@ def selected?
     def render_bidders
       return unless @bids
 
-      names = @bids.map { |bid| "#{bid.entity.name} (#{bid.price})" }
-      h(:div, "Bidders: #{names}")
-    end
+      bidders_style = {
+        'text-align': 'center'
+      }
+      sortedBids = @bids.sort_by { |bid| -bid.price }
+      names = sortedBids.map { |bid| h(:div, "#{bid.entity.name} (#{bid.price})" )}
+      h(:div, { style: bidders_style }, names )
+
+    end 
 
     def render
       onclick = lambda do
         selected_company = selected? ? nil : @company
         store(:selected_company, selected_company)
+        #TODO: Move Bid Input into Private  
       end
 
       style = {
         cursor: 'pointer',
         border: 'solid 1px rgba(0,0,0,0.2)',
-        display: 'inline-block',
+        display: 'block',
+        margin: '5px 0px'
+      }
+
+      header_style = {
+        background: 'yellow',
+        border: '1px solid',
+        'text-align': 'center',
```

Comment:
put text align center on the parent style so you don't have to specify it every time